modules/UI/videolayout/VideoContainer.js

```diff
@@ -247,19 +247,30 @@ export class VideoContainer extends LargeContainer {
      * @param {number} containerHeight container height
      * @returns {{availableWidth, availableHeight}}
      */
-    getVideoSize (containerWidth, containerHeight) {
+    getVideoSize(containerWidth, containerHeight) {
         let { width, height } = this.getStreamSize();
+
         if (this.stream && this.isScreenSharing()) {
             return getDesktopVideoSize( width,
                 height,
                 containerWidth,
                 containerHeight);
-        } else {
-            return getCameraVideoSize(  width,
-                height,
-                containerWidth,
-                containerHeight);
         }
+
+        const zoomingIndex = containerWidth / width;
+        const maximumZoomingIndex = 1.3;
+        let availableWidth = containerWidth;
+        let availableHeight = containerHeight;
+
+        if (zoomingIndex > maximumZoomingIndex) {
+            availableWidth = width * maximumZoomingIndex;
+            availableHeight = height * maximumZoomingIndex;
+        }
+
```

Comment:
Is there any reason why we don't move this part inside the getCameraVideoSize method? It looks like it's part of it.